libs/blocks/merch-twp/merch-twp.css

```diff
@@ -0,0 +1,21 @@
+@media (max-width: 1199px) {
+  .dialog-modal.twp {
+    width: 100vw;
+    height: 100vh;
```

Comment:
Have Windows or Mac devices with scrollbar enabled been considered? With scrollbars visible, this would cause vertical and horizontal scroll equal to the size of the scrollbars.

Reply:
We need to revisit this but we don't have bandwidth for TWP anymore.
I recommend to merge this as is, as we will have a throughout QA validation on mobile devices.